libcontainer/cgroups/fs/cpuacct.go

```diff
@@ -120,3 +135,42 @@ func getPercpuUsage(path string) ([]uint64, error) {
 	}
 	return percpuUsage, nil
 }
+
+func getPercpuUsageInModes(path string) ([]uint64, []uint64, error) {
+	usageKernelMode := []uint64{}
+	usageUserMode := []uint64{}
+
+	file, err := os.Open(filepath.Join(path, cgroupCpuacctUsageAll))
+	if err != nil {
+		return nil, nil, err
+	}
+	defer file.Close()
+
+	scanner := bufio.NewScanner(file)
+	scanner.Scan() //skipping header line
+
+	for scanner.Scan() {
+		lineFields := strings.SplitN(scanner.Text(), " ", cuacctUsageAllColumnsNumber)
```

Comment:
You probably should have used `cuacctUsageAllColumnsNumber + 1` as the last parameter. Otherwise you won't be able to detect a situation when there are extra fields.

Reply:
I agree, adding +1 will help if a new column appear.

Reply:
@katarzyna-z please implement ^^^